cmdb/file_export/export_types.py

```diff
@@ -26,13 +26,14 @@
 from cmdb.utils.helpers import load_class
 
 from cmdb.data_storage.database_manager import DatabaseManagerMongo
+from cmdb.framework.managers.type_manager import TypeManager
 from cmdb.framework.cmdb_object_manager import CmdbObjectManager
 from cmdb.utils import json_encoding
 from cmdb.utils.system_config import SystemConfigReader
 
-object_manager = CmdbObjectManager(database_manager=DatabaseManagerMongo(
-    **SystemConfigReader().get_all_values_from_section('Database')
-))
+database_manager = DatabaseManagerMongo(**SystemConfigReader().get_all_values_from_section('Database'))
```

Comment:
This is the point I was talking about. We have bad dependencies at many points in the software. We should solve such things in the future with DI. It would be smarter if the exporters were given the database connection in runtime. But since this was already here before and has nothing to do with the fix, this is not a break point for me.